makefiles/Mf-embed.in

```diff
@@ -0,0 +1,29 @@
+# Mf-embed.in
+# Copyright 1984-2016 Cisco Systems, Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+#               the following variables are set up by configure               #
+###############################################################################
+
+m=none
+
+InstallPrefix=/usr
+InstallLib=${InstallPrefix}/lib
```

Comment:
These variables pollute the make namespace if you include `Mf-embed` in your makefiles, so it'd be nice to simplify this all and get away from that.